다트 게임/main.py

```diff
@@ -0,0 +1,27 @@
+# 프로그래머스 - 다트게임 문제
+# https://programmers.co.kr/learn/courses/30/lessons/17682
+def solution(dartResult: str):
+    operator_dict = {
+        "S": (lambda x: x), "D": (lambda x: x ** 2),
+        "T": (lambda x: x ** 3), "#": (lambda x: x * (-1)),
+        "*": (lambda x: x * 2)
+    }
+    current_num = 0
+    answer = []
+    i = 0
+    while i <= len(dartResult):
+        if i == len(dartResult):
+            answer.append(current_num)
+            break
+        elif (ch := dartResult[i]).isnumeric():
+            answer.append(current_num)
+            if (ch2 := dartResult[i:i + 2]).isnumeric():
+                ch = ch2
+                i += 1
+            current_num = int(ch)
+        else:
+            current_num = operator_dict[ch](current_num)
+            if ch == '*':
+                answer[len(answer) - 1] = operator_dict[ch](answer[len(answer) - 1])
```

Comment:
파이썬의 마지막 요소는 항상 이렇게! ㅋㅋㄹㅃㅃ
```suggestion
                answer[-1] = operator_dict[ch](answer[-1])
```

Reply:
홀리 쉿